pyproject.toml

```diff
@@ -92,10 +92,7 @@ module = [
     "django_zero_downtime_migrations.backends.postgres.schema.*",
     "docker.*",
     "fido2.*",
-    "google.auth.*",
-    "google.cloud.*",
-    "google.resumable_media.common.*",
-    "google.rpc.status_pb2.*",
+    "google.*",
     "kombu.*",
```

Comment:
you'll want to rebase -- this was a remnant from a reverted PR

Reply:
hey Anthony! i have a smaller pr which filters out most of the relevant changes: https://github.com/getsentry/sentry/pull/70389/ to make it easier to review almost all the changes i made here is renaming stuff, with sprinkles of actual logic change. i have had few folks already volunteer their time to review the changes here, let me know if i am missing something! excited to ship this change 🚀